.github/workflows/osv-scanner-reusable.yml

```diff
@@ -16,6 +16,7 @@
 name: OSV-Scanner single vulnerability scan
 
 permissions:
+  actions: read
```

Comment:
Add the same comment here and in osv-scanner-reusable-pr.yml as well.

Reply:
added the comment